### PR TITLE
Pba 3159 eHLS asset

### PR DESCRIPTION
**Pull request by Matthew-C**

### CODE REVIEW

File: ExoPlayerSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ public void onCreate(Bundle savedInstanceState) {
     selectionMap.put( "4:3 Aspect Ratio", new PlayerSelectionOption("FwaXZjcjrkydIftLal2cq9ymQMuvjvD8", OoyalaSkinPlayerActivity.class) );
     selectionMap.put( "MP4 Video", new PlayerSelectionOption("h4aHB1ZDqV7hbmLEv4xSOx3FdUUuephx", OoyalaSkinPlayerActivity.class) );
     selectionMap.put( "HLS Video", new PlayerSelectionOption("Y1ZHB1ZDqfhCPjYYRbCEOz0GR8IsVRm1", OoyalaSkinPlayerActivity.class) );
+    selectionMap.put( "Ooyala encrypted HLS", new PlayerSelectionOption("ZtZmtmbjpLGohvF5zBLvDyWexJ70KsL-", OoyalaSkinPlayerActivity.class) );
```
Comment (lental) [inline ExoPlayerSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java]: Capital E

File: BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java [hunk 2]
```diff
@@ -52,8 +52,9 @@ public void onCreate(Bundle savedInstanceState) {
     selectionMap.put("VAST3 Ad Overlay", new PlayerSelectionOption("NuMGp1MDE6EKUv7rAmkSQ3iuLU5N5Ik0", BasicPlaybackVideoPlayerActivity.class));
     selectionMap.put("VAST3 Ad With All Of The New Events", new PlayerSelectionOption("x3MWp1MDE6yIedkirT0to1t1uy2oA8y3", BasicPlaybackVideoPlayerActivity.class));
     selectionMap.put("VAST3 AD With Icon", new PlayerSelectionOption("1qNGp1MDE6GBl7PkPlWA_FA9NJDhBp_I", BasicPlaybackVideoPlayerActivity.class));
+    selectionMap.put("VAST3 Podded with Skippable", new PlayerSelectionOption("1uNGp1MDE6DJElG8YeQJ8D7tuuJo9T-t", BasicPlaybackVideoPlayerActivity.class));
```
Comment (lental) [inline BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java]: Podded Preroll with Skippable Ad

File: BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java [hunk 3]
```diff
@@ -52,8 +52,9 @@ public void onCreate(Bundle savedInstanceState) {
     selectionMap.put("VAST3 Ad Overlay", new PlayerSelectionOption("NuMGp1MDE6EKUv7rAmkSQ3iuLU5N5Ik0", BasicPlaybackVideoPlayerActivity.class));
     selectionMap.put("VAST3 Ad With All Of The New Events", new PlayerSelectionOption("x3MWp1MDE6yIedkirT0to1t1uy2oA8y3", BasicPlaybackVideoPlayerActivity.class));
     selectionMap.put("VAST3 AD With Icon", new PlayerSelectionOption("1qNGp1MDE6GBl7PkPlWA_FA9NJDhBp_I", BasicPlaybackVideoPlayerActivity.class));
```
Comment (lental) [inline BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java]: Can you lowercase the 'd' in here for me?

File: BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java (hunk 2 above)
Comment (lental) [inline BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java]: Can you add a space before the 3 for all ads, and ad the .0? "VAST 3.0"

**Comment by Matthew-C** [comment]
Names are updated

**Pull request merged**